DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs

```diff
@@ -136,5 +136,13 @@ public override Task CreateAsync(AuditTrailCreateContext context)
             return Task.CompletedTask;
         }
 
+        private static string GetContentEventName(AuditTrailCreateContext<AuditTrailContentEvent> contentEvent, ContentApprovalPart contentApprovalPart, string reviewType)
+        {
+            contentEvent.Name = contentApprovalPart.ReviewStatus != Models.Enums.ReviewStatus.InReview
```

Comment:
Don't need to pass contentEvent, just return the string value itself without assigning it to anything.